tests/checkpointing/test_model_checkpoint.py

```diff
@@ -1273,7 +1273,7 @@ def test_none_monitor_saves_correct_best_model_path(tmpdir):
 def test_last_global_step_saved():
     # this should not save anything
     model_checkpoint = ModelCheckpoint(save_top_k=0, save_last=False, monitor="foo")
-    trainer = Mock()
+    trainer = MagicMock()
```

Comment:
I believe this shouldn't be an issue. This test was failing because `trainer.loggers` didn't have `__iter__` implemented as a `Mock`. Please let me know I should be doing something else.